maxim/logger/livekit/utils.py

```
@@ -149,3 +151,34 @@ def start_new_turn(session_info: SessionStoreEntry):
                 exc_info=True
             )
     return current_turn
+
+def extract_llm_usage(session_id: str, llm: Optional[LLM]):
```

Ruff (0.12.2)

Missing return type annotation for public function `extract_llm_usage`

(ANN201)